benchmark/bipedal-timings.cpp

```diff
@@ -4,6 +4,10 @@
 // Copyright (C) 2018-2020, LAAS-CNRS
```

Comment:
The copyright is duplicate for CNRS, and it is not in the right place.
It should be
```cpp
///////////////////////////////////////////////////////////////////////////////
// BSD 3-Clause License
//
// Copyright (C) 2018-2020, LAAS-CNRS
// Copyright (C) 2020 CTU, INRIA
// Copyright note valid unless otherwise stated in individual files.
// All rights reserved.
///////////////////////////////////////////////////////////////////////////////
```

Reply:
Thank you for giving me the format ! I didn't know what it was.